.github/workflows/publish.yaml

```diff
@@ -0,0 +1,36 @@
+name: Publish tag
+
+on:
+  # run when a pushing a tag
+  push:
+    tags:
+      - '*'
```

Comment:
can this be like "rel_?_?" so it's a little more specific ?

Reply:
This is the tag name. usually are they not in the form `v?.?.?` ? like `v1.2.3`?

Reply:
they are!   but for 15 years I've been using rel_X_Y and im old and tired

Reply:
ok, so I'll change the tag here to `v*` (`v?.?.?` does not work for >10 numbers like `v1.13.1`)

Reply:
can we add a second regexp for rel_ beacuse I am not going to be able to keep track of two different tagging formats in my projects:

```
tags: 
   - 'rel_*'
```

Reply:
my bad. I'm looking at the other project and they are all `rel_x_y_z`. Will update